Utils/src/cz/fidentis/landmarkParser/FPparser.java

```diff
@@ -80,8 +85,8 @@ public static FpModel load(String filePath) {
                 }
             }
         } catch (IOException | NumberFormatException | ParserConfigurationException | SAXException e) {
+            System.err.println(e);
```

Comment:
mozno by bolo lepsie pouzit `exceptions.printStackTrace` ci ako to je, aby sa uzivatel aspon dozvedel ze sa nieco nepodarilo